var/spack/repos/builtin/packages/singularity/package.py

```diff
@@ -136,7 +137,9 @@ def caveats(self):
         tty.warn("""
         For full functionality, you'll need to chown and chmod some files
         after installing the package.  This has security implications.
-        See: https://singularity.lbl.gov/docs-security for details.
+        For details, see: 
```

Comment:
There's a trailing whitespace here:
```suggestion
        For details, see:
```
that makes the flake8 test fail.

Reply:
Yeah I did, and I can't find it either :/ A recent security page is in the user-docs: https://sylabs.io/guides/3.3/user-guide/security_options.html?highlight=security

but it strangely seems to target the admin (with sudo) so I'm not sure if it's in the right place.